Source/WebKit/UIProcess/API/Cocoa/APISerializedScriptValueCocoa.mm

```diff
@@ -88,20 +88,21 @@ void releaseContextIfNecessary()
 
 static SharedJSContext& sharedContext()
 {
+    ASSERT(RunLoop::isMain());
     static NeverDestroyed<SharedJSContext> sharedContext;
     return sharedContext.get();
 }
 
 id SerializedScriptValue::deserialize(WebCore::SerializedScriptValue& serializedScriptValue, JSValueRef* exception)
 {
     ASSERT(RunLoop::isMain());
-    JSContext* context = sharedContext().ensureContext();
+    RetainPtr context = sharedContext().ensureContext();
 
     JSValueRef valueRef = serializedScriptValue.deserialize([context JSGlobalContextRef], exception);
     if (!valueRef)
         return nil;
 
-    JSValue *value = [JSValue valueWithJSValueRef:valueRef inContext:context];
+    JSValue *value = [JSValue valueWithJSValueRef:valueRef inContext:context.get()];
```

Comment:
Calling 'get()' here is perhaps not strictly needed?

Reply:
As far as I know, it is needed